File: Exercism Codes/Java/ResistorColorTrio.java
import java.util.List;

class ResistorColorTrio {

    private final List<String> COLORS = List.of("black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white");

    String label(String[] colors) {
        String str = COLORS.indexOf(colors[0]) + "" + COLORS.indexOf(colors[1]) + "0".repeat(COLORS.indexOf(colors[2]));
        long result = Long.parseLong(str);
        if (result >= 1000000000) {
            return (result / 1000000000) + " gigaohms";
        }
        else if (result >= 1000000) {
            return (result / 1000000) + " megaohms";
        }
        else if (result >= 1000) {
            return (result / 1000) + " kiloohms";
        }
        else {
            return result + " ohms";
        }
    }
}

File: Exercism Codes/Python/resistor_color_trio.py
def label(colors):
    dct = {
        "black": 0, "brown": 1, "red": 2, "orange": 3,
        "yellow": 4, "green": 5, "blue": 6, "violet": 7,
        "grey": 8, "white": 9
    }

    result = int(str(dct[colors[0]]) + str(dct[colors[1]]) + "0" * (dct[colors[2]]))

    if result >= 1000000000:
        return str(result // 1000000000) + " gigaohms"
    elif result >= 1000000:
        return str(result // 1000000) + " megaohms"
    elif result >= 1000:
        return str(result // 1000) + " kiloohms"
    else:
        return str(result) + " ohms"